### DISCUSSION TITLE
V0.0.7

### DISCUSSION THREAD

**Discussion by MFn233** [MFn233/PCL-Mainpage#12] [公告]
这是一次比较重要的更新，但我**好像忘记我更新了啥了**

我记得我好像要憋个大的来着，结果也忘了

~我怎么就这么健忘阿喂(╯▔皿▔)╯~

<hr /><em>This discussion was created from the release <a href='https://github.com/MFn233/PCL-Mainpage/releases/tag/V0.0.7'>V0.0.7</a>.</em>

**Comment by MFn233** [comment]
这么快就同步了？？难道我没更新？？

我记得好像加了个新闻入口来着

**Reply by SqrtTan** [reply]
没见过新闻入口:(

**Reply by SqrtTan** [reply]
我收回那句话，翻源码才看到……真是太隐蔽了
